config.json

```diff
@@ -331,9 +331,9 @@
       "difficulty": 4,
       "topics": [
         "entry_api",
-        "option",
-        "struct",
-        "vec"
+        "option_type",
```

Comment:
Same question: what's the motivation behind changing `option` to `option_type`?

Reply:
ditto: https://en.wikipedia.org/wiki/Option_type